README.md

```diff
@@ -1,13 +1,62 @@
+# A fork of `kcov` for better covering Zig
+
+This is a rather than naive fork of wonderful `kcov` for better supporting Zig projects.
+
+## It adds the ability to auto recognize Zig's `unreachable` and `@panic` for auto ignoring.
+
+Here is a screenshot of auto ignoring `unreachable`.
+
+![zig-unreachable-example](https://github.com/liyu1981/kcov/blob/master/nocover/zig-unreachable.png?raw=true)
```

Comment:
All this is nice information, but when "native" kcov supports it, I think it won't be needed (as it's shown in the output anyway).

Reply:
agree, I will remove this in later cleaned up PR.